services/QuillLMS/client/app/bundles/PremiumHub/containers/AccountManagement.tsx

```diff
@@ -298,6 +306,7 @@ export const AccountManagement: React.SFC<AccountManagementProps> = ({
     } else {
       user.role = relevantSchool.role
     }
+    user.lastSignIn = moment(user.last_sign_in).format('MMM DD, YYYY')
```

Comment:
Is there an edge case here if the user hasn't signed in yet? (e.g. The teacher record is created but the teacher has yet to sign in?)

Reply:
hmm, that's a good point-- I'll add logic to handle that just in case